lib/pdf/core/object_store.rb

```diff
@@ -96,6 +96,12 @@ def object_id_for_page(page)
         flat_page_ids = get_page_objects(pages).flatten
         flat_page_ids[page]
```

Comment:
The name of the proposed method and the intended return value don't seem to match. And this is a public API extension which I'd rather avoid.

Let's have it inline instead.

```suggestion
        pages.data[:Kids][page].identifier
```

Reply:
I completely agree with your assessment.

However, this is already a public API albeit broken because of the separation of prawn-templates. There the `get_page_objects` method is implemented. So what I did here was just to provide a dummy implementation in pdf-core to avoid exceptions when not also using prawn-templates.

So this is just really bug fixing by providing a missing implementation.

ps. I thought about in-lining but then prawn-templates may not work properly due to the more complicated page structure of real-world PDFs.

Reply:
I took a closer look at prawn-templates. `get_page_objects` is an extension point. It's private through so let's keep the method but make it private.

Reply:
I rebased onto master and made the method private.